fp_app/spec/requests/planners/planners_spec.rb

```diff
@@ -4,11 +4,34 @@
 RSpec.describe "PlannersController", type: :request do
   let(:planner) { create(:planner) }
   let(:other_planner) { create(:planner, email: "other@example.com") }
+  let(:user) { create(:user) }
 
   before do
     sign_in planner
   end
 
+  describe "planners_index" do
```

Comment:
showとかに合わせて
```suggestion
  describe "GET /planners" do
```
で良いと思います 👀

Reply:
ありがとうございます！
修正しておきます。